mkdocs.yml

```
@@ -157,6 +157,13 @@ nav:
                 - "Client certificates": mesh/grpc/ClientCertificates.md
                 - "Authorization": mesh/grpc/Authorization.md
                 - "API Specification": mesh/grpc/APISpecification.md
+          - Mesh REST API:
+                - Introduction: mesh/rest-api/introduction.md
+                - Changelog: mesh/rest-api/changelog.md
+                - API documentation: mesh/rest-api/api-documentation.md
```

```suggestion
                - API specification: mesh/rest-api/api-documentation.md
```

To be consistent with gRPC section.